contrib/client/teos_client.py

```diff
@@ -11,21 +13,20 @@
 from requests import Timeout, ConnectionError
 from requests.exceptions import MissingSchema, InvalidSchema, InvalidURL
 
-from cli.exceptions import TowerResponseError
-from cli import DEFAULT_CONF, DATA_DIR, CONF_FILE_NAME
-from cli.help import show_usage, help_add_appointment, help_get_appointment, help_register, help_get_all_appointments
-
 from common import constants
 from common.logger import get_logger, setup_logging
 import common.receipts as receipts
 from common.appointment import Appointment
 from common.config_loader import ConfigLoader
 from common.cryptographer import Cryptographer
 from common.tools import setup_data_folder
-from common.exceptions import InvalidKey, InvalidParameter, SignatureError
+from common.exceptions import InvalidKey, InvalidParameter, SignatureError, TowerResponseError
 from common.tools import is_256b_hex_str, is_locator, compute_locator, is_compressed_pk
 
-logger = get_logger()
+from . import DEFAULT_CONF, DATA_DIR, CONF_FILE_NAME
```

Comment:
Tbh I think relative imports do have an advantage here: as I expect each subfolder of /contrib to be an independent small "package", it has the advantage that it would still work even if you rename the folder.
Also, while we still want to be able to import `contrib.<stuff>` as it's useful for e2e tests, I feel like it should not be the practice, as /contrib stuff is not meant to be part of the "real" codebase. Basically, I wouldn't treat /contrib as a module throughout the codebase.

Reply:
Seems fair for the contrib bit, since as you said it'll be a standalone part only meant for developing / testing.

This does not apply for the `cli` though, where the same approach was taken.

Reply:
Just tested this as is and it does not work for me with the relative imports. Have you tested it?